### PR TITLE
Feature/pre latest

### PR DESCRIPTION
**Pull request by sbnetops**
With these changes, `polyswarmd` will use `pre-latest` images from `ci-image`-generated images and will trigger a `latest` release when releasing its own latest.

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -51,13 +51,44 @@ e2e:
...
+  only:
+    - master
+  script:
+    - >-
+      curl
+      --request POST
+      --form "token=$CI_JOB_TOKEN"
+      --form "variables[RELEASE_LATEST]=true"
+      --form ref=master
+      "https://gitlab.polyswarm.io/api/v4/projects/181/trigger/pipeline"
```
Comment (mrtizmoatwork) [inline .gitlab-ci.yml]: this is a public repo, please make the URL a CI variable

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -51,13 +51,44 @@ e2e:
...
+  only:
+    - master
+  script:
+    - >-
+      curl
+      --request POST
+      --form "token=$CI_JOB_TOKEN"
+      --form "variables[RELEASE_LATEST]=true"
+      --form ref=master
+      "https://gitlab.polyswarm.io/api/v4/projects/210/trigger/pipeline"
```
Comment (mrtizmoatwork) [inline .gitlab-ci.yml]: this is a public repo, please make the URL a CI variable

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -50,14 +50,47 @@ e2e:
   tags:
     - kube
   script:
+    - cp $E2E_COMPOSE_FILE ./docker-compose.yaml
+    - cp $E2E_CONFIG_FILE ./e2e.yaml
```
Comment (mrtizmoatwork) [inline .gitlab-ci.yml]: you'll have to test to verify, but since those are CI variables, you may instead want:

`echo $VARNAME > ./file.yaml`

Comment (sbnetops) [reply .gitlab-ci.yml]: When you set the variable type as `file` it creates a temp file with the content and sets the env var with that name to the path of the temp file. It is working as is now.

**Pull request merged**